README.md

```diff
@@ -1 +1,85 @@
-###### official website for the #TechMasters Slack Group [techmasters.chat](http://techmasters.chat)
+# Technology Masters Website
+
+The official website for the #TechMasters Slack Group [techmasters.chat](http://techmasters.chat)
+
+The site is published using [Jekyll](https://jekyllrb.com), a Ruby Gem that creates static
+pages that can be hosted anywhere.
+
+## Installation
```

Comment:
I'd rather use everything from here on as an "Alt Installation" method. with a proper "Installation" method for standard setup.